libs/core-ui/src/lib/components/AxisConfigDialog.tsx

```diff
@@ -157,24 +116,19 @@ export class AxisConfigDialog extends React.PureComponent<
           {this.state.selectedColumn.property !== cohortKey &&
             this.state.selectedColumn.property !== ColumnCategories.None && (
               <Stack>
-                {isDataColumn && (
-                  <ComboBox
-                    options={this.state.dataArray}
-                    onChange={this.setSelectedProperty}
-                    label={
-                      localization.Interpret.AxisConfigDialog.selectFeature
-                    }
-                    selectedKey={this.state.selectedColumn.property}
-                  />
-                )}
-                {isProbabilityColumn && (
-                  <ComboBox
-                    options={this.state.classArray}
-                    onChange={this.setSelectedProperty}
-                    label={localization.Interpret.AxisConfigDialog.selectClass}
-                    selectedKey={this.state.selectedColumn.property}
-                  />
-                )}
+                {isDataColumn &&
+                  this.renderComboBox(
+                    this.state.dataArray,
+                    localization.Interpret.AxisConfigDialog.selectFeature
+                  )}
+                {isProbabilityColumn &&
+                  this.renderClassComboBox(this.state.classArray)}
```

Comment:
are we missing to pass label={localization.Interpret.AxisConfigDialog.selectClass} to `this.renderComboBox`? Or is this on purpose?